Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -460,56 +461,43 @@ public override void Update(DaggerfallEntityBehaviour sender)
         // 1) In classic the initial quest can happen multiple times but this is probably a mistake.
         // 2) Additional quests will come regardless of whether the initial quest was completed or failed, as the bool is set
         //    when the initial quest begins, not on successful completion.
-        void GetVampireOrWereCreatureQuest(bool isCureQuest)
+        void StartVampireOrWereCreatureQuest(bool isCureQuest)
         {
-            // TEMP: Commenting out for now until quest deployment system is ready for this - please do not remove
-            //if (Race == Races.Vampire)
-            //{
-            //    if (isCureQuest)
-            //    {
-            //        if (DFRandom.random_range_inclusive(10, 100) < 30)
-            //            Questing.QuestMachine.Instance.InstantiateQuest("$CUREVAM");
-            //    }
-            //    else if (hasStartedInitialVampireQuest)
-            //    {
-            //        // Do a quest starting with "P0B" for player's level
-            //        // TODO: Merge with rest of quest-loading code
-            //        string[] vampireQuests = { "P0B00L01", "P0B00L03", "P0B00L04", "P0B00L06", "P0B01L02", "P0B10L07", "P0B10L08", "P0B10L10", "P0B20L09" };
-            //        List<string> vQList = new List<string>(vampireQuests);
-            //        if (DFRandom.random_range_inclusive(1, 100) < 50)
-            //        {
-            //            bool found = false;
-            //            string choice = string.Empty;
-            //            while (!found && vQList.Count > 0)
-            //            {
-            //                int index = UnityEngine.Random.Range(0, vQList.Count);
-            //                choice = vQList[index];
-            //                // 4th letter of name is the minimum level the player must be to get the quest
-            //                if (choice[3] - 48 <= level)
-            //                {
-            //                    found = true;
-            //                }
-            //                else
-            //                    vQList.RemoveAt(index);
-            //            }
-
-            //            if (found)
-            //                Questing.QuestMachine.Instance.InstantiateQuest(choice);
-            //        }
-            //    }
-            //    else if (DFRandom.random_range_inclusive(1, 100) < 50)
-            //    {
-            //        Questing.QuestMachine.Instance.InstantiateQuest("P0A01L00");
-            //        hasStartedInitialVampireQuest = true;
-            //    }
-            //}
-            ///*else
-            //{
-            //    if (playerIsWereCreature && isCureQuest && DFRandom.random_range_inclusive(1, 100) < 30)
-            //    {
-            //        Questing.QuestMachine.Instance.InstantiateQuest("$CUREWER");
-            //    }
-            //}*/
+            if (Race == Races.Vampire)
```

Comment:
Please don't uncomment this just yet. :) There is no such thing as Races.Vampire. This is done with racial overriding instead.

Reply:
Okay I commented out both calls to this method - is this okay? 
AFAICS it would simply never have done anything if race never gets set to Vampire.

Reply:
Yep, the code won't do anything because it's main condition is checking for the wrong thing. That's why I just wanted it just as reference only so it can be rewritten later. I also want it to be moved out of PlayerEntity to somewhere else, such as the effect managing vampirism itself. I'd prefer the code to stay commented out, as it's basically scheduled for deletion from this point soon.

Reply:
Okay will comment out all the code now. I agree it should be moved out of player entity, as should some of the other code at some point - it's getting a bit unwieldy as a class.

Reply:
Yeah, I'd like cleanup review of PlayerEntity at some point. But happy to let the rest sit for now. Cheers mate. :)